src/language-css/printer-postcss.js

```diff
@@ -593,7 +593,14 @@ function genericPrint(path, options, print) {
       if (isControlDirective) {
         return group(indent(concat(parts)));
       }
-
+      if (
+        atRuleAncestorNode &&
+        atRuleAncestorNode.name === "import" &&
+        node.groups[0].value === "url" &&
+        node.groups.length === 2
```

Comment:
Why two? Can you add comments about this?

Reply:
@evilebottnawi Indent is not needed for import url when node has two groups